omod/src/main/webapp/admin/concepts/conceptDrugForm.jsp

```diff
@@ -10,9 +10,8 @@
 <script type="text/javascript">
 	dojo.require("dojo.widget.openmrs.ConceptSearch");
 	dojo.require("dojo.widget.openmrs.OpenmrsPopup");
-
 	<request:existsParameter name="autoJump">
-		var autoJump = <request:parameter name="autoJump"/>;
+		var autoJump = ${openmrs:getSafeJsBoolean(param.autoJump)};
```

Comment:
I can't remember if I ever actually created the alert popup. I know that the specific example I outlined above will at least break the page's javascript on chrome. I'm almost certain I could develop  a payload that would circumvent c:out filtering to execute the alert, but it would take me another day or so.